tests/test_complex_txns.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from algokit_utils.beta.algorand_client import AlgorandClient
+
+from .transactions import get_subscribed_transactions_for_test
+
+NESTED_INNER_ROUND = 35214367
+NESTED_INNER_APP = 1390675395
+NESTED_INNER_ID = "QLYC4KMQW5RZRA7W5GYCJ4CUVWWSZKMK2V4X3XFQYSGYCJH6LI4Q/inner/5"
+
+
+@pytest.fixture()
+def algorand_mainnet() -> AlgorandClient:
+    return AlgorandClient.main_net()
+
+# TODO: Do a full snapshot assertion for both these tests. Just eyeballing it for now but it looks to line up with TS results
```

Comment:
The snapshot tests are very useful, they've picked up issues when making changes a few times, definitely recommend getting them in there in a separate PR